app/views/shared/samples/metadata/fields/_editing_field_cell.html.erb

```diff
@@ -1,7 +1,6 @@
 <td id="<%= dom_id(@sample, @field) %>">
-  <%= form_with(url: update_value_namespace_project_sample_metadata_field_path(sample_id: @sample.id), method: :patch,
+  <%= form_with(url: sample_metadatum_path(@sample, @field), method: :patch,
```

Comment:
Is this path correct? I don't see it in the routes. Maybe sample_metadata_path?

Reply:
The path is correct, if you run `bin/rails routes` it shows up near the top. In this case `metadatum` is the singular of `metadata`.